install/helm/agones/templates/controller.yaml

```diff
@@ -134,6 +134,12 @@ spec:
           value: {{ .Values.agones.image.sdk.memoryRequest | quote }}
         - name: SIDECAR_MEMORY_LIMIT
           value: {{ .Values.agones.image.sdk.memoryLimit | quote }}
+        - name: SIDECAR_RUN_AS_NONROOT
+          value: "true"
+        - name: SIDECAR_RUN_AS_USER
+          value: "1000"
```

Comment:
I suspect this value is getting ignored given the typo above.

Reply:
Yeah true